config/printer-biqu-bx-2021.cfg

```diff
@@ -0,0 +1,390 @@
+######################################################################
+# NOTE: In order to use the TFT70-BX display, you MUST add PB5 and PE5
+# to the `GPIO pins to set at micro-controller startup` section when
+# running "make menuconfig"
+######################################################################
+
+# The "make flash" command does not work on the SKR SE BX. Instead,
+# after running "make", copy the generated "out/klipper.bin" file to a
+# file named "firmware.bin" on an SD card and then restart the SKR SE BX
+# with that SD card. After klipper has been flashed once to the board,
+# you can update klipper by leaving a microSD inserted and running the
+# scripts/flash-sd.sh script.
+
+# See docs/Config_Reference.md for a description of parameters.
+
+########################################
+# Steppers
+########################################
+
+[stepper_x]
+step_pin: PG13
+dir_pin: !PG12
+enable_pin: !PG14
+microsteps: 16
+rotation_distance: 40
+full_steps_per_rotation: 400
+endstop_pin: tmc2209_stepper_x:virtual_endstop
+position_endstop: -13
+position_min: -13
+position_max: 250
+homing_speed: 30
+homing_retract_dist: 0
+
+[stepper_y]
+step_pin: PB3
+dir_pin: !PD3
+enable_pin: !PB4
+microsteps: 16
+rotation_distance: 40
+full_steps_per_rotation: 400
+endstop_pin: tmc2209_stepper_y:virtual_endstop
+position_endstop: -7
+position_min: -7
+position_max: 250
+homing_speed: 30
+homing_retract_dist: 0
+
+[stepper_z]
+step_pin: PD7
+dir_pin: PD6
+enable_pin: !PG9
+microsteps: 16
+rotation_distance: 8
+full_steps_per_rotation: 400
+endstop_pin: probe:z_virtual_endstop
+position_min: -2
+position_max: 250
+
+[stepper_z1]
+step_pin: PA8
+dir_pin: PC9
+enable_pin: !PD2
+microsteps: 16
+rotation_distance: 8
+full_steps_per_rotation: 400
+
+[extruder]
+step_pin: PC14
+dir_pin: !PC13
+enable_pin: !PC15
+microsteps: 16
+rotation_distance: 3.433
+full_steps_per_rotation: 200
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PC4
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PH4
+control: pid
+pid_Kp: 22.2
+pid_Ki: 1.08
+pid_Kd: 114
+min_temp: 0
+max_temp: 350
+pressure_advance: 0.123
+
+[safe_z_home]
+home_xy_position: 125,125
+speed: 200
+z_hop: 10
+z_hop_speed: 25
+
+########################################
+# TMC2209 configuration
+########################################
+
+[tmc2209 stepper_x]
+uart_pin: PG10
+diag_pin: ^PB11
+run_current: 0.800
+hold_current: 0.525
+sense_resistor: 0.150
+driver_SGTHRS: 127
+stealthchop_threshold: 999999
+
+[tmc2209 stepper_y]
+uart_pin: PD4
+diag_pin: ^PB12
+run_current: 0.800
+hold_current: 0.575
+sense_resistor: 0.150
+driver_SGTHRS: 137
+stealthchop_threshold: 999999
+
+[tmc2209 stepper_z]
+uart_pin: PD5
+run_current: 0.800
+hold_current: 0.525
+sense_resistor: 0.150
+stealthchop_threshold: 999999
+
+[tmc2209 stepper_z1]
+uart_pin: PC8
+run_current: 0.800
+hold_current: 0.525
+sense_resistor: 0.150
+stealthchop_threshold: 999999
+
+[tmc2209 extruder]
+uart_pin: PI8
+interpolate: false
```

Comment:
Any reason to explicitly mention `interpolate` here?

Reply:
This disabled interpolate, as the default is true